lib/internal/modules/esm/translators.js

```diff
@@ -80,7 +83,9 @@ translators.set('module', async function moduleStrategy(url) {
   debug(`Translating StandardModule ${url}`);
   const module = new ModuleWrap(url, undefined, source, 0, 0);
   moduleWrap.callbackMap.set(module, {
-    initializeImportMeta,
+    initializeImportMeta: isMain ?
+      FunctionPrototypeBind(initializeImportMeta, { isMain }) :
```

Comment:
```suggestion
     () => initializeImportMeta({ isMain }) :
```
i believe an arrow is faster than binding